src/UpdateCars.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace CarStore;
+
+use CarStore\Car;
+
+use PDO;
+
+class UpdateCars
+{
+    public PDO $db;
+
+    public function __construct($db)
+    {
+        $this->db = $db;
+    }
+
+    public function addCar(Car $car): void
+    {
+        $query = $this->db->prepare("INSERT INTO `cars`
+        (`name`, `year_made`, `zero_sixty`, `price`, `brand`, `deleted`)
+        VALUES (:car_name, :year_made, :zero_sixty, :price, :brand , 0);");
+        $query->execute([
```

Comment:
execute returns a boolean depending on whether or not the query worked, you should return that so you can check for success when calling addCar